tests/cbmc/proofs/s2n_mem_init/s2n_mem_init_harness.c

```diff
@@ -21,7 +21,7 @@
 void s2n_mem_init_harness()
 {
     /* Operation under verification. */
-    if( s2n_mem_init( ) == S2N_SUCCESS ) {
+    if(s2n_mem_init( ) == S2N_SUCCESS) {
```

Comment:
We typically space between if and parenthesis.

Reply:
Noted and done! ;)